Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM rust:alpine as builder
+
+ENV GLIBC_REPO=https://github.com/sgerrand/alpine-pkg-glibc
+ENV GLIBC_VERSION=2.35-r0
+
+RUN set -ex && apk update \
+    && apk add --no-cache ca-certificates \
+        curl bash git jq build-base linux-headers libstdc++ ca-certificates \
+    && wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub; \
+    for pkg in glibc-${GLIBC_VERSION} glibc-bin-${GLIBC_VERSION}; \
+        do curl -sSL ${GLIBC_REPO}/releases/download/${GLIBC_VERSION}/${pkg}.apk -o /tmp/${pkg}.apk; done \
+    && apk add --no-cache /tmp/*.apk \
+    && rm /tmp/*.apk
+
+WORKDIR /usr/src/foundry
+COPY . .
+
+ENV RUSTFLAGS="-C target-cpu=native"
+RUN cargo build --release
+
+RUN strip /usr/src/foundry/target/release/cast \
```

Comment:
Rust has built-in stripping since [1.59](https://blog.rust-lang.org/2022/02/24/Rust-1.59.0.html#creating-stripped-binaries), could that be used instead?

Reply:
TIL,
perhaps we add a `stripped` profile that `inherits` from `release` and sets `strip`

Reply:
Defer to whatever @tynes wants here, I think they're equivalent?